src/motion/cursor.ts

```diff
@@ -0,0 +1,33 @@
+import * as vscode from 'vscode';
+
+export default class Cursor {
```

Comment:
I'd call this class `Caret`. What's `default` for? Couldn't find it in the TS spec.